butane_core/src/many.rs

```diff
@@ -215,9 +215,15 @@ impl<T: DataObject> Default for Many<T> {
 }
 
 #[cfg(feature = "fake")]
-/// Fake data support is currently limited to empty Many relationships.
-impl<T: DataObject> Dummy<Faker> for Many<T> {
+/// Fake data support generates one value,
+/// however it is impossible to the many relationship to it.
+/// If we put them in `new_values`, we can only store the `.pk()`
+/// which makes it impossible to save the value being related to.
+impl<T: DataObject + Dummy<Faker>> Dummy<Faker> for Many<T> {
     fn dummy_with_rng<R: rand::Rng + ?Sized>(_: &Faker, _rng: &mut R) -> Self {
-        Self::new()
+        let obj = Faker.fake::<T>();
+        let ret = Self::new();
+        ret.all_values.set(vec![obj]).ok();
```

Comment:
Values in `all_values` are presumed to be already saved, and thus not savable.

The only way I can think to get this working is: a `pub fn` which loads from the db, and compares it with all parts of this `Many` to determine which DML is needed to "sync" to the db.  This could be of value if there are multiple threads, and the current thread wants to make its version of the `Many<T>` to be authoritative, despite changes that other threads may have made to the same `Many<T>`.   There are several minefields down that path.